docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       POSTGRES_USER: lrsql_user
       POSTGRES_PASSWORD: lrsql_password
       POSTGRES_DB: lrsql_db
+  clamav:
+    image: clamav/clamav:1.2.1
```

Comment:
Make sure to update the appropriate docs if you're updating the docker-compose file.